params/config.go

```diff
@@ -339,9 +339,10 @@ type ChainConfig struct {
 	// Astria Specific Configuration
 	AstriaOverrideGenesisExtraData       bool          `json:"astriaOverrideGenesisExtraData,omitempty"`
 	AstriaExtraDataOverride              hexutil.Bytes `json:"astriaExtraDataOverride,omitempty"`
-	AstriaSequencerInitialHeight         *big.Int      `json:"astriaDequencerInitialHeight,omitempty"`
-	AstriaDataAvailabilityInitialHeight  *big.Int      `json:"astriaDataAvailabilityInitialHeight,omitempty"`
-	AstriaDataAvailabilityHeightVariance *big.Int      `json:"astriaDataAvailabilityHeightVariance,omitempty"`
+	AstriaRollupName                     string        `json:"astriaRollupName,omitempty"`
+	AstriaSequencerInitialHeight         uint32        `json:"astriaDequencerInitialHeight,omitempty"`
+	AstriaDataAvailabilityInitialHeight  uint32        `json:"astriaDataAvailabilityInitialHeight,omitempty"`
```

Comment:
Two suggestions here as well:
1. Since this is a breaking change anyway, should we rename this `astriaCelestiaBaseHeight`?
2. `omitempty` has the wrong semantics as above: the marshaller would insert `0`, but that cannot be the case:
```suggestion
	AstriaCelestiaBaseHeight  uint32        `json:"astriaCelestiaBaseHeight"`
```